content/en/security/cloud_siem/detection_rules/attack_map.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ATT&CK Map
+disable_toc: false
+further_reading:
+- link: "/security/cloud_siem/detection_rules/"
+  tag: "Documentation"
+  text: "Create custom detection rules"
+---
+
+## Overview
+
+<div class="alert alert-warning">The ATT&CK Map only covers <a href="https://attack.mitre.org/matrices/enterprise/">MITRE ATT&CK Enterprise.</a></div>
+
+The MITRE ATT&CK Framework is a knowledge base used to develop specific threat models and methodologies. Use the Cloud SIEM ATT&CK Map to explore and visualize the MITRE ATT&CK Framework against Datadog's out-of-the-box rules and your custom detection rules. The ATT&CK Map displays detection rule density as a heat map to provide visibility into attacker techniques. Security teams can use the heat map to assess gaps in coverage that is relevant to their organization or team and prioritize improvements to their detection rule defenses.
+
+## View detection rules in the ATT&CK Map
+
+To view detection rules against the MITRE ATT&CK Framework:
+1. Navigate to the [Detection Rules][1] page.
+1. Click the **ATT&CK Map** button located next to **Rules List**.
+
+The default view of the map shows all Datadog out-of-the-box and custom rules for active sources, broken down into different attack techniques. Active sources are the sources of logs found and analyzed in the Cloud SIEM index.  **Note**: For the legacy SKU, all ingested logs are analyzed by Cloud SIEM unless [security filters][2] have been set up.
+
+To view the map for all sources, in the **Visualize** dropdown menu, select **All Sources**. This shows all out-of-the-box rules, including those that are not currently used to detect threats from your logs.
+
+Click the different rule density buttons to visualize the map for a specific number of rules. For example, if you click **High \+7**, only tiles that have more than seven rules enabled are shown on the map.
+
+### View a technique's information and related rules
```

Comment:
Earlier in the topic, we use both "attacker technique" and "attack technique," and here we just have "technique." I think I like the clarification of what kind of technique it is in all cases, because just "technique" on its own feels a bit broad - not sure if "attacker" vs. "attack" is a meaningful difference, though?